frontend/app/src/store/blockchain/index.ts

```diff
@@ -295,11 +295,11 @@ export const useBlockchainStore = defineStore('blockchain', () => {
     }
 
     return breakdown;
-  };
+  });
 
-  const assetBreakdown = (asset: string, chains?: string[], groupId?: string): AssetBreakdown[] => getBreakdown(asset, false, chains, groupId);
+  const assetBreakdown = (asset: string, chains?: string[], groupId?: string): ComputedRef<AssetBreakdown[]> => getBreakdown(asset, false, chains, groupId);
```

Comment:
do we expose the `getBreakdown` also to non-reactive context?

I mostly wonder if it would work if we kept the `getBreakdown` as a normal function and used `reactify` for the asset/liability breakdown functions

Reply:
I checked and we don't see to use it in a non-reactive context so it's ok